src/main/java/org/jivesoftware/openfire/plugin/passwordreset/servlet/admin/PasswordResetSettingsServlet.java

```diff
@@ -150,11 +150,10 @@ private void redirectWithMessage(
         response.sendRedirect(request.getRequestURI());
     }
 
-    @Data

```

Comment:
You're missing the toString(), hashCode() and equals() methods here. I can't remember how this is used, since you probably don't need hashCode/equals, but toString() would be useful when this is logged.

Reply:
Yeah, I think the toString()s are important.

Reply:
Should add that not every class needs every field in the toString() - but enough to identify what’s happening when examining logs.